src/options.rs

```diff
@@ -362,6 +343,23 @@ fn get_deprecated_matches(mismatch_cmd: &str) -> Option<clap::ArgMatches> {
             }
             _ => return None,
         }
+        Some(first_cmd) if first_cmd == "self" => match args.next() {
```

Comment:
Shouldn't we skip `self` fallback, because there is no release which had these working?

Reply:
Ah yes, fixed!